crates/fuel-core/src/service/genesis/importer.rs

```diff
@@ -1,15 +1,18 @@
-use self::{
-    import_task::{
-        ImportTable,
-        ImportTask,
-    },
-    progress::{
-        MultipleProgressReporter,
-        ProgressReporter,
-        Target,
-    },
+use self::import_task::{
+    ImportTable,
+    ImportTask,
 };
-use super::task_manager::TaskManager;
+
+use super::{
+    progress::MultipleProgressReporter,
+    task_manager::TaskManager,
+};
+mod import_task;
+mod off_chain;
+mod on_chain;
+use std::marker::PhantomData;
```

Comment:
Could you return the modules to the old place, please?=) Usually we have imports first, and models after